routes.js

```diff
@@ -40,25 +40,27 @@ router.get('/user-type', auth.authUser, (req, res) => userTypeController.getAll(
 router.get('/user-type/:id', auth.authUser, (req, res) => userTypeController.getById(req, res));
 
 // Company routes
-router.post('/company', auth.authAdmin, (req, res) => companyController.create(req, res));
-router.get('/company', auth.authAdmin, (req, res) => companyController.getAll(req, res));
-router.get('/company/:id', auth.authAdmin, (req, res) => companyController.getById(req, res));
+router.post('/company', (req, res) => companyController.create(req, res));
+router.get('/company', (req, res) => companyController.getAll(req, res));
+router.get('/company/:id', auth.authUser, (req, res) => companyController.getById(req, res));
 router.patch('/company', auth.authAdmin, (req, res) => companyController.update(req, res));
-router.delete('/company/:id', auth.authAdmin, (req, res) => companyController.deleteCompany(req, res));
-router.get('/company/get-by-name/:name', auth.authAdmin, (req, res) => companyController.getByNameContains(req, res));
+router.delete('/company/:id', (req, res) => companyController.deleteCompany(req, res));
+router.get('/company/get-by-name/:name', (req, res) => companyController.getByNameContains(req, res));
 
 // EventQuestion routes 
 router.post('/event-question', auth.authUser, (req, res) => eventQuestionController.create(req, res));
-router.get('/event-question', auth.authUser, (req, res) => eventQuestionController.getAll(req, res));
+// this is kinda dumb, but the way it was set up before you were getting every event question for every event 
+// in the company, it is only natural that you would get only the ones for a specific event
+router.get('/event-question/all/:id', auth.authUser, (req, res) => eventQuestionController.getAll(req, res));
```

Comment:
Change this 'id' to 'event_id' to make the path more understandable. 'id' alone would make sense if it was in relation to event question, but it is related to event. Same for other paths.